ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh

```diff
@@ -94,10 +94,10 @@ source "${SHARED_DIR}/govc.sh"
 echo "$(date -u --rfc-3339=seconds) - Extend install-config.yaml ..."
 
 # install-config for `None` platform type is expected
-declare platform_none="none: {}"
+declare platform_none="${PLATFORM_NONE_TYPE:-none}: {}"
```

Comment:
@rvanderp3 do we need to define the `platformName` too, as it is expected in the API? I think users will set it in install-config.yaml, instead of patching the infra object.

Reply:
yeah, that's a good point.  we'll have to add another environment variable for this i think.